docs/azure/sdk/logging.md

```diff
@@ -92,7 +98,7 @@ using AzureEventSourceListener listener = new AzureEventSourceListener((e, messa
 
 ## Map to ASP.NET Core logging
 
-When the <xref:Microsoft.Extensions.Azure.AzureClientServiceCollectionExtensions.AddAzureClients%2A> extension method is called, the <xref:Microsoft.Extensions.Azure.AzureEventSourceLogForwarder> service is registered. The `AzureEventSourceLogForwarder` service enables you to use the standard ASP.NET Core logging configuration for logging.
+When the <xref:Microsoft.Extensions.Azure.AzureClientServiceCollectionExtensions.AddAzureClients%2A> or the <xref:Microsoft.Extensions.Azure.AzureClientServiceCollectionExtensions.AddAzureClientsCore%2A> extension method is called, the <xref:Microsoft.Extensions.Azure.AzureEventSourceLogForwarder> service is registered. The `AzureEventSourceLogForwarder` service enables you to use the standard ASP.NET Core logging configuration for logging.
```

Comment:
I'm on the fence here with whether to mention `AddAzureClientsCore` explicitly or just call out that `AzureEventSourceLogForwarder` can be registered as a singleton if, for some odd reason, you prefer not to register your SDK clients via `AddAzureClients` - which is the recommended approach.

Thoughts?

Reply:
I don't have a strong opinion on AddAzureClientsCore vs registering AzureEventSourceLogForwarder, but I think it is a good idea to try to call out that it is unusual to use either of these two approaches as opposed to just using AddAzureClients.

Reply:
.... unless you're not actually using explicit clients.   😄    _(which is an uncommon corner case, admittedly)_

Reply:
Yeah, I guess we can provide an example where someone might do this, e.g. your app doesn't actually use any Azure SDK clients but it uses some other package that uses the Azure SDK under the hood, such as the data protection extensions.

Reply:
For as long as `AddAzureClientsCore` remains on the public API surface, we need to document its purpose. I like Josh's recommendation of providing an example. I'll add this.